docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
<p>Add something like "as you can see the results with optimization_level=3 is closer to the ideal distribution due to reduced errors from lower number of CX".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>